org.mixedrealitytoolkit.input/InteractionModes/INearInteractionModeTarget.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Mixed Reality Toolkit Contributors
+// Licensed under the BSD 3-Clause
+
+using UnityEngine;
+using UnityEngine.XR.Interaction.Toolkit;
+
+namespace MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// This interface is used to update the enable state of the Components that are in the <see cref="ProximityEnabledComponents"/>
+    /// array (set in Editor) and to keep track of which <see cref="Collider"/> and <see cref="XRBaseInteractor"/> duples are triggering proximity.
+    /// </summary>
+    /// <remarks>
+    /// This interface is needed to prevent a circular reference between MRTK Input and MRTK UX Core Scripts packages.
+    /// </remarks>
+    public interface INearInteractionModeTarget
```

Comment:
(sorry of the not suggesting this sooner)....

However, we should rename this to `IProximityHoverInteractable` to match the patterns of XRI (e.g. `IXRHoverInteractable`

Reply:
Got it, renamed in latest commit.